### ISSUE TITLE
nn.ConvTranspose3d with output_padding

### ISSUE THREAD

**Issue by azad96** [wang-xinyu/tensorrtx#1055]
I have an input of size [1, 64, 6, 15, 20], and when I apply torch's ConvTranspose3d on it with the following arguments

ConvTranspose3d(64, 32, kernel_size=(3, 3, 3), stride=(2, 2, 2), padding=(1, 1, 1), output_padding=(0, 0, 0), bias=False)

I get an output of size [1, 32, 11, 29, 39]

I managed to get the same output in TensorRT with IDeconvolutionLayer as follows:

auto deconv1 = network->addDeconvolutionNd(*input, 32, Dims{3, {3, 3, 3}}, deconv1_weights, empty_weights);
assert(deconv1);
deconv1->setStrideNd(nvinfer1::Dims{3, {2, 2, 2}});
deconv1->setPaddingNd(nvinfer1::Dims{3, {1, 1, 1}});

However, I have a ConvTranspose3d layer with output_padding=(1, 1, 1), which results in an output of size [1, 32, 12, 30, 40]. IDeconvolutionLayer has some methods namely setPrePadding, setPostPadding, setPaddingMode. I tried using these to get the same output shape but they did not make any difference.

My question is whether output_padding is supported in TensorRT and if so how to use it?

**Comment by wang-xinyu** [comment]
This is TensorRT question, not related to this repo, please reach out to NVIDIA or check the documentation.
